### DISCUSSION TITLE
Volume levels and Panic Buttons

### DISCUSSION THREAD

**Discussion by garamoniker** [GeekFunkLabs/fluidpatcher#63] [General]
Hi guys!

I've been having an amazing time with the Squishbox. (Thanks so much Bill!) I've got my first live gig with the rig coming up in mid November and I'm looking forward to using it. 

Just 2 things: Some Sounfonts have volume levels way higher than others - I was wondering if there's any way to preset higher or lower starting volume through router rules on particular soundfonts?

The 2nd thing: Is there away to create a panic button to kill sticky midi notes that can occur from time to time?

Thanks, guys!

**Comment by albedozero** [comment]
Sure! There are ways of solving both problems.

1st thing:

You can't limit volume on a specific soundfont, but you can use that soundfont only on specific channel(s) and limit their volume using a MIDI expression (CC#11) message. The volume on a channel is the product of both MIDI volume (CC#7) and expression, so you can use one to set the max level on the channel, and the other to make dynamic changes while you play. [This section of lesson 4](https://youtu.be/K_TKv0kQubU?t=174) demonstrates if this isn't clear. My preference is to use CC7 to set levels on each channel, CC11 to do dynamics, and route my keyboard's volume slider to the `synth.gain` fluidsetting to give me a global volume, like so:
```yaml
- {type: cc, chan: 1, par1: 7, fluidsetting: synth.gain, par2: 0-127=0.1-1.2}
```

2nd thing:

You can route a control (probably a pad/button) to send an ALL NOTES OFF (CC#123) message with a value of 64 or greater on whatever channel(s) you're playing. This will tell all notes on those channels to go into release phase. You could do ALL SOUND OFF (CC#120), but that will abruptly cut off sounds, which could sound bad. Sometimes I don't want to end all my notes, either - just the stuck ones - so my solution is to quickly tap the stuck notes again (assuming I can tell which ones they are), which usually sends the missing note off. The sticky notes seem to me to happen more with some MIDI controllers than others - I wonder if anyone else has this same experience?

**Reply by garamoniker** [reply]
Perfect. Thanks, Bill. Definitely had a look at 4 and tried it out by essentially increasing synth.gain on all patches except the ones that were loud, essentially bringing my quiet patches up to match the louder patch.

Thanks for your help, as always, Bill.

I've found sticky notes seem to have more to do with the the usb cable/socket in question. I notoriously had this problem running Mainstage into my Edirol PCR800 (hence my panic button obsession) and then I switched cables and it went away...for a while. It mainly only occurs if the USB moves about too much with the Squishbox. I used to think it happened because I played too many notes, too quickly but it's obviously more about physical movement, itself. I get a little animated when I play so this tends to happen sometimes. I've run a vortex 2 on wireless and had very few sticky problems unless I push the range and wander off.
